### PR TITLE
CB-7798 tests has failed with a generic "Cloudera Manager [Template i…

### PR DESCRIPTION
**Pull request by akanto**
…nstall] operation failed" error message.

See detailed description in the commit message.

### CODE REVIEW

File: cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/ClouderaManagerPollingServiceProvider.java [hunk 1]
```diff
@@ -131,60 +135,61 @@ public PollingResult startPollingCmHostDecommissioning(Stack stack, ApiClient ap
...
             AbstractClouderaManagerCommandCheckerTask<ClouderaManagerPollerObject> listenerTask) {
         ClouderaManagerPollerObject clouderaManagerPollerObject = new ClouderaManagerPollerObject(stack, apiClient, commandId);
-        return clouderaManagerCommandPollerObjectPollingService.pollWithTimeoutSingleFailure(
+        return clouderaManagerCommandPollerObjectPollingService.pollWithAbsoluteTimeout(
                 listenerTask,
                 clouderaManagerPollerObject,
                 POLL_INTERVAL,
-                numAttempts);
+                maximumWaitTimeInSeconds,
+                TOLERATE_FIVE_CONSECUTIVE_ERRORS).getLeft();
```
Comment (pdarvasi) [inline cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/ClouderaManagerPollingServiceProvider.java:192]: this is used in one place, maybe we should consider passing it as parameter

File: cloud-common/src/main/java/com/sequenceiq/cloudbreak/polling/PollingService.java [hunk 2]
```diff
@@ -21,37 +21,43 @@
...
             }
-            if (failures >= maxFailure) {
-                LOGGER.debug("Polling failure reached the limit which was {}, poller will drop the last exception.", maxFailure);
+            if (consecutiveFailures >= maxConsecutiveFailures) {
+                LOGGER.debug("Polling failure reached the limit which was {}, poller will drop the last exception.", maxConsecutiveFailures);
                 statusCheckerTask.handleException(actual);
                 return new ImmutablePair<>(PollingResult.FAILURE, actual);
             } else if (success) {
                 LOGGER.debug(statusCheckerTask.successMessage(t));
+                LOGGER.debug("Set the number of consecutive failures to 0, since we resived a positve answer. Original number of consecutiveFailures: {}",
```
Comment (pdarvasi) [inline cloud-common/src/main/java/com/sequenceiq/cloudbreak/polling/PollingService.java:58]: typo: resived ->  received

File: cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/ClouderaManagerPollingServiceProvider.java (hunk 1 above)
Comment (akanto) [reply cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/ClouderaManagerPollingServiceProvider.java]: Thanks

File: cloud-common/src/main/java/com/sequenceiq/cloudbreak/polling/PollingService.java (hunk 2 above)
Comment (akanto) [reply cloud-common/src/main/java/com/sequenceiq/cloudbreak/polling/PollingService.java]: Thanks.

**Pull request merged**